package.json

```diff
@@ -34,9 +34,9 @@
     "stage:venia": "yarn venia run start",
     "stats:venia": "yarn venia run build:analyze && yarn venia run stats",
     "storybook:venia": "yarn venia run storybook",
-    "test": "jest",
-    "test:ci": "jest --no-cache -i --json --outputFile=test-results.json",
-    "test:debug": "node --inspect-brk node_modules/.bin/jest -i --no-cache",
+    "test": "node --experimental-worker node_modules/.bin/jest",
```

Comment:
`--experimental-worker` is needed because jest runs on `node` and `MessageChannel` is only available if `--experimental-worker` flag is passed while running jest.